src/scss/utility/colors.scss

```diff
@@ -0,0 +1,42 @@
+@use '../mixins/colors.scss';
+
+.kd-theme-darkstone-80 {
```

Comment:
For the utility classes. let's use a double dash for the modifier. Ex: `.kd-theme--darkstone-80`

Reply:
> For the utility classes. let's use a double dash for the modifier. Ex: `.kd-theme--darkstone-80`

Ah, the BEM standard. Ok.

Reply:
Just include this change, then we're good to merge IMO. Follow up on themes later.